hack/jenkins/build.sh

```diff
@@ -62,7 +62,11 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+if [[ "${KUBE_FASTBUILD:-}" == "true" ]]; then
```

Comment:
`make quick-release` just does `KUBE_RELEASE_RUN_TESTS=n KUBE_FASTBUILD=true build/release.sh`.

I suspect we're maybe fixing this the wrong way.